tests/fixtures/arm64.output.service.json

```diff
@@ -24,7 +24,7 @@
       ],
       "handler": "newrelic-lambda-wrapper.handler",
       "layers": [
-        "arn:aws:lambda:us-east-1:451483290750:layer:NewRelicNodeJS12XARM64:31"
+        "arn:aws:lambda:us-east-1:451483290750:layer:NewRelicNodeJS12XARM64:33"
```

Comment:
would it make edited these files easier if version were a variable?

Reply:
@yahmed153 The jest-based fixture tests...are hard to turn into something that'd be variable-friendly. Part of the issue here is the non-deterministic layer-version spec in AWS. Getting rid of that would be tricky, since "get the latest layer for this runtime" is an essential part of this plugin's operation.